### PR TITLE
Remove admin bar

### PR DESCRIPTION
**Pull request by bookernath**
#### What?

The admin bar will be moved to platform JS so it can be evolved separately from the theme. Removing it from cornerstone. Should not be merged until platform JS solution is in place.

#### Tickets / Documentation

- [MERC-6628](https://jira.bigcommerce.com/browse/MERC-6628)

### CODE REVIEW

**Comment by fthobe** [comment]
@bookernath Hey, we run a very customized theme, could you elaborate on the impact and if we should do something on our side?

**Comment by bookernath** [comment]
@fthobe this admin bar is a feature you see when you're viewing your own storefront as a control panel user. It makes it easier to understand what state your storefront is in (live/maintenance/prelaunch) and also makes it easier to launch page builder from any page of the storefront. We're moving it out of the theme into a platform-injected script, so this PR removes it from the concerns of theme developers.

If anything, this should just make your life a little bit easier and cut some JS out of your shopper-facing bundle... but if you never pull in this update, it shouldn't really matter.

File: templates/layout/base.html [hunk 1]
```diff
@@ -46,12 +46,9 @@
         {{~inject 'productSize' theme_settings.product_size}}
         {{~inject 'showAdminBar' theme_settings.show-admin-bar}}
```
Comment (bc-dhuynh) [inline templates/layout/base.html:47]: This can probably go away too right?

Comment (bookernath) [reply templates/layout/base.html]: Nice catch!

**Comment by bc-dhuynh** [comment]
@bookernath Sorry dude, last one https://github.com/bookernath/cornerstone/blob/144103a226f282aa14a44e8b61fe091bba226713/config.json#L346 that I can think of anyways

**Comment by bookernath** [comment]
@bc-dhuynh fixed. Thanks!

**Review by bc-dhuynh — APPROVED** [review]

**Review by yurytut1993 — APPROVED** [review]
LGTM, but can you please also update changelog by the structure fix/feat(storefront): ticketId short explanation message

**Review by bc-alexsaiannyi — APPROVED** [review]

**Comment by BC-tymurbiedukhin** [comment]
Should we update adminBar style vars in assets/scss/settings/stencil/maintenanceNotice/_settings.scss?

**Comment by BC-tymurbiedukhin** [comment]
Should we remove adminBar from page builder as well? (schema + schemaTranslations files)

**Comment by bookernath** [comment]
Thanks @BC-tymurbiedukhin for your review, I've made those updates ♻️

File: assets/scss/settings/stencil/maintenanceNotice/_settings.scss [hunk 2]
```diff
@@ -14,11 +14,3 @@ $maintenanceNotice-z-index:             zIndex("highest");
 
 $maintenanceNotice-header-font-family:  stencilFontFamily("headings-font");
 $maintenanceNotice-header-margin:       0 0 spacing("half");
-
-$adminBar-height:                       46px;
-$adminBar-color:                        color("whites", "bright");
-$adminBar-logo-color:                   #273a8a;
-$adminBar-contentLink-color:            #3E67F8;
-$adminBar-contentLink-hover-color:      #f0f3fe;
-$adminBar-tooltip-bg-backgroundColor:   #313440;
```
Comment (BC-tymurbiedukhin) [inline assets/scss/settings/stencil/maintenanceNotice/_settings.scss:23]: Do we use this var in _focus-tooltip.scss file?

**Pull request closed without merging**